pkg/installer/v1dot18.go

```diff
@@ -0,0 +1,181 @@
+// This file is part of MinIO Direct CSI
+// Copyright (c) 2021 MinIO, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package installer
+
+import (
+	"context"
+)
+
+type v1dot18 struct{}
+
+// installers
+func installNSV1dot18(ctx context.Context, config *InstallConfig) error {
```

Comment:
Make these functions as method to `v1dot18` so that it won't be accessible outside of the structure.